clients/drcachesim/tools/opcode_mix.h

```diff
@@ -83,7 +84,7 @@ class opcode_mix_t : public analysis_tool_t {
 
 protected:
     struct worker_data_t {
-        std::unordered_map<app_pc, int> opcode_cache;
+        std::unordered_map<app_pc, std::pair<int, uint>> opcode_category_cache;
```

Comment:
Hmm I was going to say `s/uint/dr_instr_category_t/` but this combines them.  Add a comment on why it's a uint and not the enum type?

Reply:
Done, I changed it to a more meaningful structure as suggested.